méli-mélo/2024-02-charts/demo.html

```diff
@@ -0,0 +1,465 @@
+---
+title: Charts
+dateModified: 2024-02-23
+description: "Create a visual representation of tabular data."
+lang: en
+breadcrumbs: [
+	{ title: "Charts", link: "méli-mélo/2024-02-charts/index.html" }
+]
+script:
+- js/deps/chart.min.js
+- js/charts.js
+---
+
+<div class="wb-prettify all-pre hide"></div>
+
+<h2>Line chart</h2>
+<table class="text-right table table-condensed table-bordered table-striped table-hover gc-chart provisional" data-chart-type="line" data-chart-options='{
+	"elements": {
+		"point": {
+			"radius": 0
+		}
+	}
+}'>
+	<caption>Account holdings by type (line chart)</caption>
+	<thead>
+		<tr>
+			<th class="text-left">Years</th>
+			<th class="text-right" data-chart-set-options='{
+				"borderColor": "#28667a",
+				"backgroundColor": "#28667a",
+				"fill": true
+			}'>Amounts contributed or transferred</th>
+			<th class="text-right" data-chart-set-ignore="true">Investment income</th>
+			<th class="text-right" data-chart-set-options='{
+				"borderColor": "#e58a65",
+				"backgroundColor": "#e58a65",
+				"fill": true
+			}'>Total account holdings</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<th data-chart-value="Year 1">1</th>
```

Comment:
Comment/suggestion: I think the attribute `data-chart-value` is also used to provide the numeral value for the data cell  and here it is more about the label (header cell).
I would like to suggest the following notation instead
```suggestion
			<th><span class="wb-inv">Year </span>1</th>
	<!-- Where the label is extracted from the "textContent" DOM property of the <th> element -->
```

or

```suggestion
			<th data-chart-label="Year 1">1</th>
```

Reply:
I will keep as is se we have an example of the data-chart-value option. Otherwise, this is a great suggestion.